src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Find your way with field data in the Web Vitals extension
+subhead: The Web Vitals extension now shows you where your local experiences are in relation to
+  real user experiences in the field.
+description: The Web Vitals extension now shows you where your local experiences are in relation
+  to real user experiences in the field.
+authors:
+  - rviscomi
+date: 2021-06-30
+hero: image/STd8eW8CSiNp5B1bX0R6Dww2eH32/KwcWeCZ6LoIfTLRgVxp4.png
+alt: The Milky Way galaxy with a pin labeled You Are Here
+tags:
+  - blog
+  - web-vitals
+  - chrome-ux-report
+  - tools
+---
+
+[Core Web Vitals](https://web.dev/vitals) are the most important metrics you should be measuring to
+understand your users' experiences, because when users have good experiences,
+[good things happen](https://wpostats.com/)! The [Web Vitals extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma?hl=en)
+for Chrome is one tool in the [Web Vitals toolbox](https://web.dev/vitals-tools/) that focuses on
+giving you performance data about the pages you visit as you browse the web. But you may be
+wondering, "how typical are user experiences like mine?" The last thing we want to do is perpetuate
+the myth that _if it's fast for me, it's fast for thee_. So we're launching version 1.0 of the
+Web Vitals extension with a new feature to integrate real-user data from the
+[Chrome UX Report](https://web.dev/chrome-ux-report/) (CrUX) with your local Core Web Vitals
+experiences, putting them in the greater context of how other users have experienced the same pages
+and websites. It comes with a sparkly new UI and I'm excited to show you how it works.
```

Comment:
```suggestion
[Core Web Vitals](/vitals) are the most important metrics to measure to
understand your users' experiences, because when users have good experiences,
[good things happen](https://wpostats.com/)! The [Web Vitals extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma?hl=en)
for Chrome is one tool in the [Web Vitals toolbox](/vitals-tools/) that shows you 
performance data about the pages you visit as you browse the web. 

Page performance depends on many factors, including hardware and network quality. How
you experience a page on your machine may be very different from how majority of users
experience it. That's why the version 1.0 of the Web Vitals extension includes a new feature
that integrates real-user data from the [Chrome UX Report](/chrome-ux-report/) (CrUX) 
with your local Core Web Vitals measurements. This feature displays your local measurements
in the context of how other users have experienced the same pages. It comes with
a sparkly new UI and I'm excited to show you how it works.
```

Reply:
- Chunking paragraphs for readability
- Replaced some instances of _experience_ (nebulous) with _measurement_ (concrete)
- Rewrote the rhetorical and "me/thee" part to be simpler and straightforward

Please edit the suggestions if I changed the meaning somewhere, but maintain the simpler style.

Reply:
I've also trimmed "pages and websites" to pages. I know that's simplistic, if you think it's important to be precise in the intro just edit it back in.